changelogs/refactor-payment-recommendations-eligibiility

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: Tweak
+Comment: This is a slight refactor on how the recommendations on the payment settings are displayed. I extracted the eligibility part to a separate component, to be reused in an upcoming PR.
```

Comment:
Thanks for using the changelogger. We just have a lint that also checks that you mentioned the PR number in the comment, so you may wanna tweak that.